tests/data/test_nrrd_reader.py

```
@@ -128,6 +145,32 @@ def test_read_with_header_index_order_c(self, data_shape, filename, expected_sha
+            filepath = os.path.join(tempdir, filename)
+            nrrd.write(filepath, test_image, header=reference_header)
+            reader = NrrdReader()
+            image_array, image_header = reader.get_data(reader.read(filepath))
+        self.assertIsInstance(image_array, np.ndarray)
+        self.assertEqual(image_array.dtype, dtype)
+        self.assertTupleEqual(image_array.shape, data_shape)
+        # spatial_shape must exclude the channel axis
+        self.assertTupleEqual(tuple(image_header["spatial_shape"]), data_shape[1:])
+        # channel dim 0 must be identified
+        self.assertEqual(image_header["original_channel_dim"], 0)
+        # affine must be a valid 4×4 matrix (3 spatial dims → 4×4)
```

Ruff (0.15.15)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)